### PR TITLE
File Added : Assignment 6 & 9

### PR DESCRIPTION
**Pull request by urmilagagare**

### CODE REVIEW

File: urmila/JAVASCRIPT/Assignment9/palindrome.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                padding: 7px 11px;
+                margin: 0 8px;
+                font-weight: bold;
+                font-size: 17px;
+                border: 1px solid gray;
+                border-radius: 4px;
+            }
+        </style>
+    </head>
+    <br><br>
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment9/palindrome.html]: why this br tags between head and body tag

File: urmila/JAVASCRIPT/Assignment9/prime_number2.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                padding: 7px 11px;
+                margin: 0 8px;
+                font-weight: bold;
+                font-size: 17px;
+                border: 1px solid gray;
+                border-radius: 4px;
+            }
+        </style>
+    </head>
+    <br>
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment9/prime_number2.html]: Remove this br it is useless

File: urmila/JAVASCRIPT/Assignment9/prime_palindrome3.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+                padding: 7px 11px;
+                margin: 0 8px;
+                font-weight: bold;
+                font-size: 17px;
+                border: 1px solid gray;
+                border-radius: 4px;
+            }
+        </style>
+    </head>
+    <br><br>
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment9/prime_palindrome3.html]: remove unused br tags

File: urmila/JAVASCRIPT/Assignment9/string_palindrome.html [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+                padding: 7px 11px;
+                margin: 0 8px;
+                font-weight: bold;
+                font-size: 17px;
+                border: 1px solid gray;
+                border-radius: 4px;
+            }
+        </style>
+    </head>
+    <br>
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment9/string_palindrome.html]: Remove this br tag

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

**Review by Ashwin2812 — APPROVED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by vijaya1811 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**